mode_client/clients.py

```diff
@@ -224,6 +225,48 @@ def delete(self, space: str) -> None:
         self.request("DELETE", f"/spaces/{space}")
 
 
+class ModeDefinitionClient(ModeBaseClient):
+    def get(self, definition_token: str) -> Definition:
+        response = self.request("GET", f"/definitions/{definition_token}")
+
+        return Definition.parse_obj(response)
+
+    def list(
+        self, filter_: Optional[str] = None, tokens: Optional[List[str]] = None
+    ) -> List[Definition]:
+        params = {"filter": filter_, "tokens": tokens}
+        response = self.request("GET", "/definitions", params=params)
+        definitions = response["_embedded"]["definitions"]
+
+        return parse_obj_as(List[Definition], definitions)
+
+    def create(self, data: Dict[str, Any]) -> Definition:
```

Comment:
I am not sure what definition's data is required to create definition objects as the API documentation does not specify it and this endpoint returns 404 in some cases (same for the update method)
However the main purpose is to add github sync definition, when we use `list`, `get` and `sync` endpoint so it is not a blocker.

Reply:
then how about we just remove these methods(create, update, delete)? reason being i don't think we will create definition via code? since mostly the definitions are created and used by analysts and are created in Mode UI

Reply:
don't have strong opinion here tho, just think we're probably not gonna use these create, update and delete functions in our mode-dashboard repo